packages/grafana-ui/src/components/IconButton/IconButton.story.tsx

```diff
@@ -30,7 +30,8 @@ const meta: Meta<typeof IconButton> = {
     tooltip: 'sample tooltip message',
     tooltipPlacement: 'top',
     variant: 'secondary',
-    ariaLabel: 'sample aria-label content',
+    ariaLabel: 'this property is deprecated',
```

Comment:
I think if we are telling people not to use this anymore we should remove it from the controls. They should be able to see the deprecation warning on the props list.

Also, I think this `aria-label` will never be used because if we are setting a `tooltip` prop (even if it is an empty string), it always goes through that `if`. Maybe we should have another story for `IconButton` without `tooltip` prop? Otherwise we could have to have a more complex control here that deletes the tooltip prop?

Reply:
Done.